app/scripts/directives/sf-aloha-format-style.js

```diff
@@ -17,6 +17,19 @@ angular.module('authoringEnvironmentApp').directive(
                 );
                 element.toggleClass('disabled', isDisabled);
                 element.bind('click', function () {
+
+                    // this block sets the Aloha.activeEditable
+                    // to the editable of the current Aloha.Selection
+                    // needed due to a bug with the formatBlock function
```

Comment:
possibly, but I think upgrading aloha would be a pretty big undertaking, and I think would require us to refactor all of the custom aloha plugins we have created (nsimage, snippets, undo, and quotes).